tests/createTestNiftis.py

```diff
@@ -47,7 +47,7 @@ def createNiftiTestFiles(shouldValidate: bool = True):
     Create base 3D NIfTI1 file all others are created from
     """
     convertDicomFileToNifti(test_dicomPath, test_3DNifti1Path)
-    nifti1_3D = nib.load(test_3DNifti1Path)
+    nifti1_3D = nib.load(test_3DNifti1Path, False)
```

Comment:
Do you need to say 'mmap=False'? I'm confused why just passing False works.

Could you put your PR description of the issue as a comment above this line?